.husky/post-merge

```diff
@@ -3,10 +3,16 @@
 
 # The hook documentation: https://git-scm.com/docs/githooks.html#_post_merge
 
+# Refresh dependencies when pulled changes might change the deps.
 changedManifests=$( ( git diff --name-only HEAD ORIG_HEAD  | grep -E '(package.json|pnpm-lock.yaml|pnpm-workspace.yaml|composer.json|composer.lock)$' ) || echo '' )
 if [ -n "$changedManifests" ]; then
 	printf "It was a change in the following file(s) - refreshing dependencies:\n"
 	printf "    %s\n" $changedManifests
 
 	pnpm install --frozen-lockfile
 fi
+
+# Cleanup .wireit cache that is older than 4 weeks; otherwise, the repository directory size keeps growing and growing.
+find plugins/*/.wireit/* -maxdepth 0 -type d -ctime +28 -exec rm -rf {} \;
+find packages/js/*/.wireit/* -maxdepth 0 -type d -ctime +28 -exec rm -rf {} \;
+find plugins/woocommerce/client/*/.wireit/* -maxdepth 0 -type d -ctime +28 -exec rm -rf {} \;
```

Comment:
Should we leave some messages here, at the start and end of the cache removal script explaining what is happening?

Reply:
@kalessil here is the comment, looks like I didn't submit it 😅

Reply:
Makes sense: pushing ceee320beb60e3f36c0f392322ea675193d92c3e, which prompts about the cache cleanup (prompt before cleanup should be enough - don't want to overload console output).

Reply:
I'll re-test the changes on MacOS/Linux and merge afterwards.

Reply:
Thanks! I tested the updated scripts manually and it works great on my end.